node-operators/hyperdrive/disaster-recovery-and-node-migration.md

```diff
@@ -38,10 +38,60 @@ Wallet recovery is the process for restoring a node from an offline seed phrase
 To minimize downtime, you should initiate a wallet recovery on new hardware **as soon as you realize the old filesystem is not recoverable.**
 {% endhint %}
 
-First, [install Hyperdrive as usual](installation.md) on your new node.&#x20;
+First, [install Hyperdrive as usual](installation.md) on your new node. When Hyperdrive asks you to create a wallet, reply `n`. This will save you the step of creating and validating a new mnemonic which won't be used. 
 
 Next, use the `hyperdrive wallet recover` command to regenerate your node wallet based on your seed phrase backup. From here, you should review your configuration with `hyperdrive service config` and [monitor the node as usual](monitoring-your-node.md) to ensure it is operational.
 
+### Recovering the StakeWise Module
+
+After you've recovered your wallet, you can enable the StakeWise module in the Hyperdrive Configuration terminal `hyperdrive service config` -> Modules menu.
+
+After exiting the configuration terminal, Hyperdrive asks you to verify you don't have any running validators, and a final verification that if you did have validators running, you've waited 15 minutes, because Hyperdrive can't determine if you attested in the last 15 minutes. Because you're about to regenerate the same keys, ensure that the node you're recovering hasn't attested in the last 15 minutes.
```

Comment:
```suggestion
After exiting the configuration terminal, Hyperdrive asks you to verify you don't have any running validators, and a final verification that if you did have validators running, you've waited 15 minutes, because Hyperdrive can't determine if you attested in the last 15 minutes. Because you're about to regenerate the same keys, this will apply to you, and **you MUST ensure that the node you're recovering hasn't attested in the last 15 minutes or you may be slashed!**
```